core/src/main/resources/template/simple/script.ftl

```diff
@@ -0,0 +1,54 @@
+<#--
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+-->
+<script nonce="${parameters.nonce}"<#rt/>
+<#if parameters.async?has_content>
+ async="${parameters.async}"<#rt/>
+</#if>
+<#if parameters.charset?has_content>
+ charset="${parameters.charset}"<#rt/>
+</#if>
+<#if parameters.defer?has_content>
+ defer="${parameters.defer}"<#rt/>
+</#if>
+<#if parameters.src?has_content>
+ src="${parameters.src}"<#rt/>
+</#if>
+<#if parameters.type?has_content>
+ type="${parameters.type}"<#rt/>
+</#if>
+<#if parameters.name?has_content>
+ name="${parameters.name}"<#rt/>
+</#if>
+<#if parameters.referrerpolicy?has_content>
+ referrerpolicy="${parameters.referrerpolicy}"<#rt/>
+</#if>
+<#if parameters.nomodule?has_content>
+ nomodule="${parameters.nomodule}"<#rt/>
```

Comment:
we need to change this to check if the value is set to true, and add the keyword `nomodule` only, instead of `nomodule="${parameters.referrerpolicy}"`

Reply:
![image](https://user-images.githubusercontent.com/44476526/88059136-1a386a00-cb6d-11ea-8708-67781234ac05.png)
so you mean that right?

Reply:
I think so, we might also want to check if the content it has is `"true"` but I'm not entirely sure!

Reply:
also pasting this link [for HTML spec](https://html.spec.whatwg.org/multipage/scripting.html) for reference on which attributes need this change. It seems like async, defer and nomodule are booleans
![image](https://user-images.githubusercontent.com/35872165/88059906-2cff6e80-cb6e-11ea-803c-0ced07a72294.png)

Reply:
Fixed